sherpa/astro/ui/tests/test_astro_ui_io.py

```diff
@@ -29,6 +29,9 @@
 from sherpa.astro import ui
 from sherpa.astro.data import DataPHA
 from sherpa.astro.instrument import ARF1D, RMF1D
+from sherpa.utils.err import ArgumentErr
+from sherpa.utils.testing import requires_data, requires_fits, \
```

Comment:
Line 28 - which I can't comment on in this interface, is also a `from sherpa.utils.testing` line and can be removed (left over from a less-than-judicious rebase).